src/dolphin/install/mainlineInstallation.ts

```diff
@@ -0,0 +1,298 @@
+import type { SyncedDolphinSettings } from "@dolphin/config/config";
+import { addGamePath, getSlippiMainlineSettings, setSlippiMainlineSettings } from "@dolphin/config/config";
+import { IniFile } from "@dolphin/config/iniFile";
+import { spawnSync } from "child_process";
+import { app } from "electron";
+import electronLog from "electron-log";
+import * as fs from "fs-extra";
+import os from "os";
+import path from "path";
+import { lt } from "semver";
+import semverRegex from "semver-regex";
+
+import type { DolphinInstallation } from "../types";
+import { DolphinLaunchType } from "../types";
+import { downloadLatestDolphin } from "./download";
+import type { DolphinVersionResponse } from "./fetchLatestVersion";
+
+const log = electronLog.scope("dolphin/installation");
```

Comment:
```suggestion
const log = electronLog.scope("dolphin/mainlineInstallation");
```
Do you think this would be helpful?